dotcom-rendering/src/components/Discussion/Comment.tsx

```diff
@@ -19,6 +19,7 @@ import { Column } from './Column';
 import { RecommendationCount } from './RecommendationCount';
 import { Row } from './Row';
 import { Timestamp } from './Timestamp';
+import { DispatchContext } from '../Discussion';
```

Comment:
You may want to put `DispatchContext` in its own file, as I think there's a circular import happening here: `Discussion` transitively imports `Comment`, and `Comment` imports `Discussion`?

This is suggested in the docs[^1]. You may find it necessary to move the `Action` type into this file as well, as it's needed to parameterise `createContext`.

[^1]: https://react.dev/learn/scaling-up-with-reducer-and-context#step-1-create-the-context